src/plugins/data_views/server/rest_api_routes/runtime_fields/create_runtime_field.ts

```diff
@@ -49,14 +49,11 @@ export const createRuntimeField = async ({
     throw new Error(`Field [name = ${name}] already exists.`);
```

Comment:
Seems like this check doesn't work for composite fields

Reply:
Fixed and added integration test to verify